examples/k8s_deploy/README.md

```diff
@@ -0,0 +1,97 @@
+# Deploying a Kubernetes cluster on Lambda in 1-click with SkyPilot
+
```

Comment:
We should update this readme to point to lambda's official guide once that is published.